BunnyHopMaster/Packages/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "com.oneleif.debugconsole": "https://github.com/oneleif/UnityDebugConsole.git",
+    "com.oneleif.debugconsole": "file:D:/unity/github repo/UnityDebugConsole/UnityDebugConsole/Assets/com.oneleif.debugconsole",
```

Comment:
we don't want to store the on-disk location. I don't have this location so it breaks the import

Reply:
I'll fix that